src/component/entity-form.js

```diff
@@ -0,0 +1,28 @@
+import {customElement, bindable} from 'aurelia-framework';
+import {resolvedView} from 'aurelia-view-manager';
+
+@resolvedView('spoonx/form', 'entity-form')
+@customElement('entity-form')
+export class EntityForm {
+
+  @bindable entity;
+
+  @bindable behavior;
+
+  @bindable skip = [];
+
+  attached() {
+    let types = this.entity.getMeta().metadata.types;
```

Comment:
We can also set the meta on this, and not populate this.elements. Now there are two loops.

```html
<form-group repeat.for="type of types"
  value.bind="entity[type]"
  label.bind="type"
  element.bind="types[type]" />
```